### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by SchumErik** [holzschu/Carnets#92]
Great application!! For visualization it would be great to integrate voila support, if possible from a technology perspective (runs on jupyter lab)

**Comment by holzschu** [comment]
Thanks for the suggestion. Just to be sure, you are talking about this voila: https://github.com/voila-dashboards/voila ?
Also, have you tried with "pip install voila"? What happens? (most likely it fails, but if it works...)

**Comment by SchumErik** [comment]
Yes, thats the one. When I tried to install it failed with requirement for jupyter-core>=4.6.0. Would be fantastic if you could add - it would allow for simple creation of dashboards. Especially helpful is the preview module available in jupyter lab. Based on your earlier comments I infer that you plan to also implement Jupyter lab which would be fantastic...  :)

**Comment by ifuchs** [comment]
There is also Panel (similar in many ways to Voila).

**Comment by holzschu** [comment]
Hi, 
"!pip install voila" now works on the latest TestFlight version (soon to become the AppStore version).
Interestingly, it also installs "jupyter-server", which apparently was not needed to run Jupyter notebooks.
I have no idea if it will actually work; keep in mind that there is little parallelism on the iPad.
